README.md

```diff
@@ -40,7 +47,7 @@ fi
 The script aims to provide a complete and functioning GNS3 experience. Using the script is simple. Recommended usage is to clone the repo and to execute the script via bash.
 
 ```bash
-git clone https://github.com/SirToffski/Arch-GNS3-Helper.git
+git clone https://github.com/LaumiH/Arch-GNS3-Helper.git
```

Comment:
This change should only apply to your forked repo. On my master the link should point to `https://github.com/SirToffski/Arch-GNS3-Helper.git`